internal/provider/services/test_helper.go

```diff
@@ -15,3 +16,8 @@ func MustMarshallString(value string, t *testing.T) types.AttributeValue {
 		return result
 	}
 }
+
+func ResolveS3Locations(providerID, providerVersion, value string) string {
```

Comment:
Why is this function in a file called `test_helper.go` if we're using it in the real code? Suggest moving to a better location!

Reply:
Shifted this to storage.go